front/style/_committee.scss

```diff
@@ -330,6 +330,17 @@
                 }
             }
         }
+
+        &-description {
```

Comment:
Yes, the comment is at: https://github.com/EnMarche/en-marche.fr/pull/581#discussion_r106326343.